config.yaml.sample

```diff
@@ -174,6 +174,18 @@ metatile:
   # to a metatile-based system from an individual format based one.
   store_metatile_and_originals: false
 
+# Configuration for where to store the tiles of interest set
+toi:
+  # We support storing the TOI in S3 or as a file
+  type: <file or s3>
+  file:
+    # The name of the file to store the TOI
+    name: toi.txt.gz
+  s3:
+    # The bucket and key for the s3 object that will contain the tiles of interest
+    bucket: test_bucket
+    key: toi/toi.txt.gz
```

Comment:
:+1: thanks!

I like that you namespaced the configuration based on type. I wish I had done it that way for the other configuration sections instead of flattening it out, feels cleaner.